plugins/redis/redis.cpp

```diff
@@ -266,10 +266,16 @@ void Connection::parseOptions(ICodeContext * ctx, const char * _options)
         }
     }
 }
-void Connection::resetContextErr()
+void Connection::reset(ICodeContext * ctx, const char * password, unsigned _timeout)
 {
-    if (context)
-        context->err = REDIS_OK;
+    updateTimeout(_timeout);
```

Comment:
This will need to be ```timeout.reset(_timeout);``` when upmerged to 6.0.0, not sure on how such things are supposed to be handled, please inform.